### DISCUSSION TITLE
Stability of this package?

### DISCUSSION THREAD

**Discussion by williamboman** [termux/termux-language-server#10] [Q&A] [answered]
Hello! I maintain the https://github.com/mason-org/mason-registry/ package registry which provides editor-related tooling (e.g. LSP servers). Due to the alpha version and being relatively new; I was wondering what the stability of this package is and if you'd consider it stable enough to be added to such registries?

Thanks!

**Comment by Freed-Wu** [comment, marked as answer]
Thanks for your discussion. I believe the APIs of this language server will not change heavily in the future. So if any downstream project decide to use it, it will be welcome.

> I maintain the https://github.com/mason-org/mason-registry/ package

Currently, I use <https://github.com/neoclide/coc.nvim> so I am not familiar with mason. However, I heard it have a great effect on the plugin ecosystem written in lua. I am happy to see this project attract your interest and wish mason will be better.

**Reply by Conarius** [reply]
Quick clarification:
mason.nvim and the mason-registry are for downloading and installing development tools such as LSP servers like this one.
When it comes to attaching and LSP server to a buffer that would go to nvim-lspconfig. Maybe there could be a pr added to that as well.
In the mason registry there is a pr for making this available to download and install through mason. That's why William, the maintainer of mason.nvim and the mason-registry asked for clarification about how stable it is and how actively maintained it is.
